app/src/main/java/com/example/tryggaklassenpod/languages/abouts/AboutSwe.kt

```diff
@@ -1,2 +1,121 @@
 package com.example.tryggaklassenpod.languages.abouts
 
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.em
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import com.example.tryggaklassenpod.R
+import com.example.tryggaklassenpod.ui.theme.md_theme_light_primary
+
+
+@Composable
+fun AboutSwe(navController: NavController){
+
+    Column (
+        modifier = Modifier.fillMaxWidth()
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Spacer(modifier = Modifier.padding(10.dp))
+
+        Text(text = "Trygga Klassen Podcasten är din viktigaste källa för insikter som förbättrar " +
+                "välmåendet för barn och unga. Genom engagerande samtal med experter behandlar vi " +
+                "kritiska ämnen som missbruk, mobbning, och mental hälsa, samt ämnen som rör " +
```

Comment:
A lot of duplicate code with the language parts.
Would make more sense to send the necessary text from  res/strings to the reusable language component

Reply:
Another reason I wanted to merge before I continue so it won't affect the master!!! But apparently I have to deal with something that has nothing to do with me :|

Reply:
please check your code and let me know if there is anything affecting your part